.github/workflows/automatic_code_review.yml

```diff
@@ -0,0 +1,81 @@
+name: Automatic code review
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  review_code:
+    name: Run Danger 
```

Comment:
Should this job name `Code review by Danger` instead? 

We might have another code review flow by another tool later (such as [Sonarqube](https://github.com/marketplace/actions/sonarqube-scan)), and keeping it as `Run Danger` will conflict to the last step `Run Danger` at the bottom 💭

Reply:
sounds like a reasonable cause for changing the name @luongvo, by the way the Danger doesn't just review code, it also review other part of the pull request as well, and code is a part of the pull request, so I think naming it to `Pull request review by Danger` is better 😁

Reply:
@minhnimble I keep my thumbs for you on this